src/components/product-card/product-card.tsx

```diff
@@ -28,11 +23,9 @@ export const ProductCard = ({
             <div className={styles.cardContent}>
                 <p className={styles.description}>{name}</p>
                 {price?.formatted?.price && (
-                    <Price
-                        fullPrice={price?.formatted?.price}
-                        discountedPrice={price?.formatted?.discountedPrice}
-                    />
+                    <Price fullPrice={price?.formatted?.price} discountedPrice={price?.formatted?.discountedPrice} />
```

Comment:
it is a question do we want to show price for out of stock items, but it is more product decision

Reply:
I have recently searched one product on few stores and it really depends on a store, whether they show old price if product is out of stock. I would say 50/50